### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by stevenchung213** [wallabyjs/quokka#392]
### Issue description or question
Not sure if an update caused this but I just updated quokka to v1.0.185 via Webstorm Plugins > Update and then started receiving this output.

UPDATE**: I did update Node from 12.2 to 12.5 earlier as well.
### Sample code

```javascript
Upon creation of a new scratch file or opening an existing scratch file
```

### Sample repository link

all existing/new scratch files display the output below

### Quokka.js Console Output

```
Quokka v1.0.185
Webstorm v2019.1.3

**************************************************************************
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:107:11)
    at Object.basename (path.js:668:5)
    at WebSocketServer.emit (events.js:200:13)
    at Server.emit (events.js:200:13)
    at onParserExecuteCommon (_http_server.js:585:14)
    at onParserExecute (_http_server.js:525:3)
```

### Code editor version

Webstorm v2019.1.3

### OS name and version

Windows 10

**Comment by ArtemGovorov** [comment]
Thanks for reporting the issue. It is fixed in v1.0.186.

**Comment by stevenchung213** [comment]
YOU ROCK ARTEM!  GO WALLABY TEAM!
